src/gso.ts

```diff
@@ -147,24 +168,25 @@ export class GSO {
     const gsoState = await this.state(projectName);
 
     const [soAuthority, soAuthorityBump] = await web3.PublicKey.findProgramAddress(
-      [
-        Buffer.from(utils.bytes.utf8.encode('gso')),
-        gsoState.toBuffer(),
-      ],
+      [Buffer.from(utils.bytes.utf8.encode('gso')), gsoState.toBuffer()],
       this.program.programId,
     );
     const so = new StakingOptions(this.connection.rpcEndpoint);
 
     const soState = await so.state(`GSO${projectName}`, baseMint);
     const soBaseVault = await so.baseVault(`GSO${projectName}`, baseMint);
-    const soOptionMint = await so.soMint(strikeAtomsPerLot, `GSO${projectName}`, baseMint);
+    const soOptionMint = await so.soMint(
```

Comment:
why all these formatting changes? not a problem, but not necessary. if you want to do all these changes, generally should be doing them in a separate pull request.